plugins/woocommerce-blocks/assets/js/blocks/product-filters-overlay/block.json

```diff
@@ -32,6 +32,19 @@
 		"overlayPosition": {
 			"type": "string",
 			"default": "left"
+		},
```

Comment:
Does adding the style here not affect the frontend? The gap issue identify from before is in the editor.

Reply:
@roykho Yes, it applies to the frontend, but I don't see a reason why we shouldn't match the editor style with the frontend style.

Reply:
So the fix originally was just for the editor as commented from here https://github.com/woocommerce/woocommerce/pull/48472#pullrequestreview-2116811799

I don't know if this fix is needed in the front end or not as we're not currently working on the frontend.